src/System.IO.Pipes/tests/System.IO.Pipes.Tests.builds

```diff
@@ -7,6 +7,15 @@
     </Project>
     <Project Include="System.IO.Pipes.Tests.csproj">
       <OSGroup>Windows_NT</OSGroup>
+    </Project>
+    <Project Include="System.IO.Pipes.Tests.csproj">
+      <OSGroup>Unix</OSGroup>      
+      <TargetGroup>netstandard1.3</TargetGroup>
```

Comment:
Small nit: We are trying to keep this metadata order consistent across the repo, can you please reorder like: TargetGroup, OsGroup, TestTFMs?